tasks/managed/update-component-sbom/update-component-sbom.yaml

```diff
@@ -124,7 +124,7 @@ spec:
         - name: sourceDataArtifact
           value: $(params.sourceDataArtifact)
     - name: update-sboms
-      image: quay.io/konflux-ci/release-service-utils:8cb945feb15ca453a90b2316da7a69e3ecd4f845
+      image: quay.io/konflux-ci/release-service-utils:2d8b7b558cfb95506eb2fc79ef82332878bb6a75
```

Comment:
Why did change? Any specific change in the utils image you were looking for? Or maybe done by accident?